Zend/zend_vm_gen.php

```diff
@@ -19,7 +19,7 @@
 	 $Id$
 */
 
-$header_text = <<< DATA
+const HEADER_TEXT = <<< DATA
```

Comment:
Presumably the latest stable version since internals contributors are the ones that will be using it. :)